### PR TITLE
Add fifth section

### PR DESCRIPTION
**Pull request by Flasto4ka**
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -386,3 +386,60 @@ ul {
...
+  margin-bottom: 40px;
+}
+
+.information__block {
+  display: flex;
+  gap: 8px;
+}
+
+.information__image {
+width: 162px;
```
Comment (flast1k) [inline css/style.css:420]: format

File: index.html [hunk 2]
```diff
@@ -264,6 +265,70 @@ <h1 class="banner__title">
...
+              </div>
+            </div>
+            <div class="column-lg-4"></div>
+          </div>
+
+          <div class="row">
+            <div class="information__block">
+              <div class="column-lg-3">
+                <div class="information__part">
+                  <img class="information__image" src="img/Теричев Максим.svg" alt="" />
```
Comment (flast1k) [inline index.html:289]: add alt

**Review by flast1k — APPROVED** [review]

File: css/style.css (hunk 1 above)
Comment (Flasto4ka) [reply css/style.css]: +

File: index.html (hunk 2 above)
Comment (Flasto4ka) [reply index.html]: +

**Pull request merged**